add version to setup.cfg
remove "dirty-tag" from setuptools-scm
relax version requiments.txt 

author: John Dey Fred Hutch
diff -ruN gdc-client-2.3.orig/pyproject.toml gdc-client-2.3/pyproject.toml
--- gdc-client-2.3.orig/pyproject.toml	2024-09-11 10:22:54.000000000 -0700
+++ gdc-client-2.3/pyproject.toml	2026-02-12 12:26:37.720567000 -0800
@@ -38,10 +38,6 @@
 py_version = 38
 skip = [".gitignore", ".dockerignore"]
 
-[tool.setuptools_scm]
-local_scheme = "dirty-tag"
-
-
 [tool.pytest.ini_options]
 minversion = 6.0
 python_files = "test_*.py"
diff -ruN gdc-client-2.3.orig/requirements.txt gdc-client-2.3/requirements.txt
--- gdc-client-2.3.orig/requirements.txt	2024-09-11 10:22:54.000000000 -0700
+++ gdc-client-2.3/requirements.txt	2026-02-12 12:29:53.026791000 -0800
@@ -4,59 +4,59 @@
 #
 #    pip-compile
 #
-attrs==23.2.0
+attrs>=23.2.0
     # via
     #   jsonschema
     #   referencing
-certifi==2024.7.4
+certifi>=2024.7.4
     # via requests
-charset-normalizer==3.3.2
+charset-normalizer>=3.3.2
     # via requests
-idna==2.8
+idna>=2.8
     # via requests
-importlib-metadata==7.0.1
+importlib-metadata>=7.0.1
     # via gdc-client (pyproject.toml)
-importlib-resources==6.4.0
+importlib-resources>=6.4.0
     # via
     #   jsonschema
     #   jsonschema-specifications
-intervaltree==3.0.2
+intervaltree>=3.0.2
     # via gdc-client (pyproject.toml)
-jsonschema==4.23.0
+jsonschema>=4.23.0
     # via gdc-client (pyproject.toml)
-jsonschema-specifications==2023.12.1
+jsonschema-specifications>=2023.12.1
     # via jsonschema
-lxml==5.2.2
+lxml>=5.2.2
     # via gdc-client (pyproject.toml)
-pkgutil-resolve-name==1.3.10
+pkgutil-resolve-name>=1.3.10
     # via jsonschema
-progressbar2==3.55.0
+progressbar2>=3.55.0
     # via gdc-client (pyproject.toml)
-python-utils==2.7.1
+python-utils>=2.7.1
     # via progressbar2
-pyyaml==6.0.1
+pyyaml>=6.0.1
     # via gdc-client (pyproject.toml)
-referencing==0.35.1
+referencing>=0.35.1
     # via
     #   jsonschema
     #   jsonschema-specifications
-requests==2.32.3
+requests>=2.32.3
     # via gdc-client (pyproject.toml)
-rpds-py==0.19.1
+rpds-py>=0.19.1
     # via
     #   jsonschema
     #   referencing
-six==1.16.0
+six>=1.16.0
     # via
     #   progressbar2
     #   python-utils
-sortedcontainers==2.4.0
+sortedcontainers>=2.4.0
     # via intervaltree
-termcolor==1.1.0
+termcolor>=1.1.0
     # via gdc-client (pyproject.toml)
-urllib3==1.25.11
+urllib3>=1.25.11
     # via requests
-zipp==3.17.0
+zipp>=3.17.0
     # via
     #   importlib-metadata
     #   importlib-resources
diff -ruN gdc-client-2.3.orig/setup.cfg gdc-client-2.3/setup.cfg
--- gdc-client-2.3.orig/setup.cfg	2024-09-11 10:22:54.000000000 -0700
+++ gdc-client-2.3/setup.cfg	2026-02-12 15:31:15.265154000 -0800
@@ -1,5 +1,6 @@
 [metadata]
 name = gdc-client
+version = 2.3
 description = The gdc-client provides several convenience functions over the GDC API which provides general download/upload via HTTPS.
 long_description = file: README.md
 long_description_content_type = text/markdown
@@ -27,8 +28,8 @@
 install_requires =
     jsonschema
     lxml
-    PyYAML>=6.0.1
-    intervaltree~=3.0.2
+    PyYAML>=6.0.0
+    intervaltree>=3.0.2
     importlib_metadata
     termcolor~=1.1.0
     requests
